### PR TITLE
Debug hotfix

### PR DESCRIPTION
**Pull request by scribam**
With imgui v1.67, it is now illegal to call `Begin("")` with an empty string. See "Breaking Changes" in https://github.com/ocornut/imgui/releases/tag/v1.67
The debug build asserts without the commit "gui: Add window name for game selector":
```
Vita3K: /home/scribam/CLionProjects/Vita3K/src/external/imgui/imgui.cpp:4746: bool ImGui::Begin(const char*, bool*, ImGuiWindowFlags): Assertion `name != __null && name[0] != '\0'' failed.
```

### CODE REVIEW

**Comment by VelocityRa** [comment]
Thanks!

File: src/external/CMakeLists.txt [hunk 1]
```diff
@@ -23,14 +23,17 @@ endif()
 
 option(CAPSTONE_BUILD_SHARED "Build shared library" OFF)
 option(CAPSTONE_BUILD_TESTS "Build tests" OFF)
-option(CAPSTONE_ARM_SUPPORT "ARM support" ON)
```
Comment (Zangetsu38) [inline src/external/CMakeLists.txt]: With remove this line if you want on it ?

Comment (scribam) [reply src/external/CMakeLists.txt]: Because the default value is ON

Comment (scribam) [reply src/external/CMakeLists.txt]: See https://github.com/aquynh/capstone/blob/master/CMakeLists.txt#L40

Comment (VelocityRa) [reply src/external/CMakeLists.txt]: because it's on by default
I agree thought, shouldn't have been removed probably

Comment (Zangetsu38) [reply src/external/CMakeLists.txt]: yes i know, just me i leave it here ^^

**Pull request merged**